### DISCUSSION TITLE
Version 2023.4.2

### DISCUSSION THREAD

**Discussion by tregan01** [tesla-android/issue-tracker#210] [General]
I see the latest version is 2023.4.2, however the instructions say after extracting/downloading the zip file, "using raspberry Pi Imager flash an image named: "tesla-android-2022.45.1-single-image.img" to your SD card". I'm assuming that's just an oversight and I should be installing 2023.4.2, correct?

Thanks!

**Comment by mikegapinski** [comment]
Yes, it's a typo

**Reply by tregan01** [reply]
Thank you!
